homeassistant/components/snapcast/__init__.py

```diff
@@ -1 +1,54 @@
-"""The snapcast component."""
+"""Snapcast Integration."""
+from dataclasses import dataclass
+import logging
+import socket
+
+import snapcast.control
+
+from homeassistant.components.media_player import MediaPlayerEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST, CONF_PORT
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+
+from .const import DOMAIN, PLATFORMS
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@dataclass
+class HomeAssistantSnapcastData:
+    """Snapcast data stored in the Home Assistant data object."""
+
+    server: snapcast.control.Snapserver
+    clients: list[MediaPlayerEntity]
+    groups: list[MediaPlayerEntity]
```

Comment:
In the future this will be extended to a full snapcast class handling callbacks from the server (add/remove entities, setting availablity). Is models.py still a good file name?

Reply:
OK. Right now it is simply called `HomeAssistantSnapcast`. I could put it in a snapcast.py module? Or I could go with the wrapper suggestion, both fine for me.